cli/http_util.rs

```diff
@@ -123,11 +124,18 @@ pub async fn fetch_once(
 
   if response.status().is_client_error() || response.status().is_server_error()
   {
-    let err = generic_error(format!(
-      "Import '{}' failed: {}",
-      args.url,
-      response.status()
-    ));
+    let err = if response.status() == StatusCode::NOT_FOUND {
```

Comment:
While this was done for the sniffing that was refactored out, it sill makes sense IMO, that we can distinguish between 404 Not Found errors and other response errors, like we can with local files.  Lumping everything in a generic error seems wrong.